README.md

```diff
@@ -19,3 +19,10 @@ Make sure you have golang installed
 deploy`
 7. To verify deployed project:
 `curl https://go-whiteboard.appspot.com/helloWorld`
+
+## Writing and Compiling .proto files into .go
+1. Create your go file in a directory under `src`
```

Comment:
Could you add steps for installing protoc and go plugin:
- https://developers.google.com/protocol-buffers/docs/downloads
- go install google.golang.org/protobuf/cmd/protoc-gen-go@latest

Reply:
Also: "Create your go file" => "Create your proto file"